src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/BaseHandlerTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.LanguageServerClient.Razor.Logging;
+using Moq;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp
+{
+    public abstract class BaseHandlerTest
+    {
+        public BaseHandlerTest()
+        {
+            var logger = new Mock<ILogger>(MockBehavior.Strict).Object;
```

Comment:
Might just be easier to create a `TestLogger : ILogger` otherwise you need to maintain all of these various log expectations where in reality we don't really care about whta's logged at test time